content/docs/esc/access-management/oidc-authentication.md

```diff
@@ -0,0 +1,16 @@
+---
+title_tag: Configure OpenID Connect authentication | Pulumi ESC
+meta_desc: This page provides an overview of how Pulumi ESC can be configured for OIDC authentication with a trusted identity provider.
+title: Configure OpenID authentication with ESC
+h1: Configure OpenID Connect authentication
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  esc:
+    name: OIDC authentication
+    parent: pulumi-esc-access-management
+    weight: 3
+---
+
+Pulumi supports secure authentication by integrating with trusted external identity providers using OpenID Connect (OIDC). When configured as an OIDC client, Pulumi establishes a trust relationship with third-party providers such as Google, AWS or GitHub to accept and validate their issued OIDC tokens. After validation, these tokens are exchanged for short-lived Pulumi access tokens, which removes the need for hardcoded credentials.
```

Comment:
We have OIDC connections from ESC to various clouds, and we have inbound OIDC from a cloud or CI/CD. How can we better disambiguate what we are talking about here?  Can we include some scenarios with links to our other content like EKS running in AWS using ESC go get secrets, or GitHub running CI/CD getting access to AWS through Pulumi ESC?